XRTK.Oculus/Packages/com.xrtk.oculus/Runtime/Providers/BoundarySystem/OculusBoundaryDataProvider.cs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using UnityEngine;
+using XRTK.Attributes;
+using XRTK.Definitions;
+using XRTK.Definitions.BoundarySystem;
+using XRTK.Interfaces.BoundarySystem;
+using XRTK.Oculus.Extensions;
+using XRTK.Oculus.Plugins;
+using XRTK.Services;
+
+namespace XRTK.Oculus.Providers.BoundarySystem
+{
+    [RuntimePlatform(typeof(OculusPlatform))]
+    [System.Runtime.InteropServices.Guid("8EF0CAB5-A37C-4912-AD5E-1E57E92A314D")]
+    public class OculusBoundaryDataProvider : BaseDataProvider, IMixedRealityBoundaryDataProvider
+    {
+        /// <inheritdoc />
+        public OculusBoundaryDataProvider(string name, uint priority, BaseMixedRealityProfile profile, IMixedRealityBoundarySystem parentService)
+            : base(name, priority, profile, parentService)
+        {
+            boundarySystem = parentService;
+        }
+
+        private readonly IMixedRealityBoundarySystem boundarySystem;
+
+        /// <inheritdoc />
+        public override void Enable()
+        {
+            base.Enable();
+
+            boundarySystem.SetupBoundary(this);
+        }
+
+        #region IMixedRealityBoundaryDataProvider Implementation
+
+        /// <inheritdoc />
+        public BoundaryVisibility Visibility => BoundaryVisibility.Visible;
```

Comment:
If there is no API to query the current visibility state for boundary this should be "Unknown".

Reply:
Can you clarify what the purpose of testing "visibility" is?  Couldn't see any comment documentation to state what it expects from the API.
Is it:

* Is the boundary currently visible?
* Is there a boundary to be visible?
* Has a boundary been configured?

Just so I know what to hook it up to.

Reply:
Implementation updated